hr_employee_firstname/tests/test_hr_employee_firstname.py

```diff
@@ -5,7 +5,7 @@
 import odoo
 from odoo.exceptions import ValidationError
 from odoo.tests.common import TransactionCase
-from odoo.tools import submap
+from odoo.tools.misc import submap
```

Comment:
Why this change?

I couldn't find any related change that doesn't allow the previous syntax, and I think importing methods directly from `odoo.tools` is preferred.

If really required, it might be a good idea to document it in the commit description.

Reply:
When I ran the tests to verify that they worked fine I got this error:

 ```
File "/home/odoo/instance/extra_addons/hr/hr_employee_firstname/tests/__init__.py", line 3, in <module>
    from . import test_hr_employee_firstname
  File "/home/odoo/instance/extra_addons/hr/hr_employee_firstname/tests/test_hr_employee_firstname.py", line 8, in <module>
    from odoo.tools import submap
ImportError: cannot import name 'submap' from 'odoo.tools' (/home/odoo/instance/odoo/odoo/tools/__init__.py)
```

I tried to import the tools library using `from odoo import tools` and then `tools.submap` but got the same error.